runner_service/configuration.py

```diff
@@ -55,6 +55,12 @@ def __init__(self, mode='dev'):
             "env/ssh_key"
         )
 
+        # maximum age of an artifact folder in days
+        self.artifacts_remove_age = 30
```

Comment:
This implies that always the removal of older artifacts is set... what happens if i do not want to remove old executions? 
Yes... i can set "artifacts_remove_age" to 32000 for example... but i will have a thread that is not going to do nothing...

I think that setting "artifacts_remove_age=0" must disable the removal of old artifacts folder.